packages/calcite-components/src/tests/commonTests.ts

```diff
@@ -918,7 +918,7 @@ async function getTagAndPage(componentTestSetup: ComponentTestSetup): Promise<Ta
  * });
  *
  * @param {ComponentTestSetup} componentTestSetup - A component tag, html, or the tag and e2e page for setting up a test.
- * @param {DisabledOptions} [options={ focusTarget: "host" }] - Disabled options.
```

Comment:
This looks like an autofix, but the file itself looks unrelated to these changes. Let's exclude it from this PR and then do a full lint sweep afterwards.